TAF/utils/scripts/manual/deploy-edgex.sh

```diff
@@ -0,0 +1,39 @@
+# This script is to deploy KVM and get IP address of qemu
+
+#!/bin/bash
+
+GLOBAL_VAR_DIR=${WORK_DIR}/TAF/config/global_variables.py
+
+if [ ${MODULE} ==  "KVM" ] || [ ${MODULE} ==  "kvm" ]
+then
+    cd ${WORK_DIR}/../edge-linux-test-pytest
+    python3 -m pip install -r ${HOME}/edge-linux-test-pytest/requirements.txt
+
+    if [ -f kvm_data.json ]
```

Comment:
From what I can tell, we are assuming that if this file exists, the KVM is running.  You delete this file between runs, so that helps.  That said, it is not a guarantee that the KVM is running, especially if a person runs this test locally.

Is there a way to check that the KVM is actually running when it finds the file?  For instance, pull out the PID and check the server to see if that PID is still actively running?

Reply:
yes , using below command we can check that "start_kvm.py" process is running or not.

"**ps ax | grep -v grep | grep start_kvm.py**"

So, the flow will be like if "kvm_data.json" file is there then will check that "start_kvm.py" process is running or not. and if process is running then will not deploy edgex again.